libraries/AP_Camera/AP_Camera.cpp

```diff
@@ -251,16 +301,44 @@ MAV_RESULT AP_Camera::handle_command_long(const mavlink_command_long_t &packet)
         }
         return MAV_RESULT_DENIED;
     case MAV_CMD_IMAGE_START_CAPTURE:
-        if (!is_zero(packet.param2) || !is_equal(packet.param3, 1.0f) || !is_zero(packet.param4)) {
+        // currently we don't handle the parameter 4 i.e. sequence number
```

Comment:
Let's change this to a positive comment which will be more informative.  So for example, "// check interval and total images"